src/test/java/core/basesyntax/service/WriteDataToFileTest.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.FruitDto;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class WriteDataToFileTest {
+    private static WriteDataToFile writer;
+    private static ReadService readService = new ReadService();
+    private static final String toFileName = "src/test/resources/currentReportText.csv";
+    private final String currentReportText = "fruit,quantity" + System.lineSeparator()
+            + "apple,150" + System.lineSeparator() + "banana,100";
+
+    @BeforeClass
+    public static void beforeClass() {
+        writer = new WriteDataToFile();
+    }
+
+    @Test
+    public void writer_data_ok() throws IOException {

```

Comment:
```suggestion
    public void writer_validData_ok() {
```
Better put readString method in try-catch block